### DISCUSSION TITLE
Notification service for COMPASS

### DISCUSSION THREAD

**Discussion by aman1esmagico** [Sunbird-Lern/Community#168] [Query] [answered]
Hi team, 
We are creating different modules of compass like marketplace, WPCAS and passbook. We have a requirement of notifying users at multiple level. So, can you please tell, what we are using for notification service and their documentation to send email to the user. 

Help will be appreciated

**Comment by Jayaprakash8887** [comment]
@aman1esmagico Kindly follow https://lern.sunbird.org/use/developer-guide/notification-service

**Comment by mohitga** [comment, marked as answer]
Hi @aman1esmagico, did above answer helpful and solved your query?
